web-app/js/contacts.js

```diff
@@ -372,6 +375,7 @@ $(function () {
             .done(function(data) {
                 $icon.removeClass("fa-spinner fa-spin")
                 $icon.addClass("fa-check-circle")
+                return true;
```

Comment:
Asyn method. It doesn't returns nothing. There isn't any function waiting for this response.